src/ClosureExporter.php

```diff
@@ -63,7 +63,7 @@ public function export(Closure $closure, int $level = 0): string
 
         --$start;
         $uses = $this->useStatementParser->fromFile($fileName);
-        $tokens = token_get_all('<?php ' . implode('', array_slice($fileContent, $start, $end - $start)));
+        $tokens = token_get_all('<?php ' . PHP_EOL . implode('', array_slice($fileContent, $start, $end - $start)));
```

Comment:
We should always use \n because \r\n isn't required, inconsistent and doesn't play well with Docker (in PHP it is tolerable but with sh it basically doesn't work).